src/containers/SecretsModal/SecretsModal.js

```diff
@@ -425,6 +434,28 @@ export /* istanbul ignore next */ class SecretsModal extends Component {
             onRemove={this.handleRemove}
             onAdd={this.handleAdd}
           />
+          {errorMessage &&
+            (open && (
+              <InlineNotification
+                kind="error"
+                title={intl.formatMessage({
+                  id: 'dashboard.secretsModal.error.title',
+                  defaultMessage: 'Error:'
+                })}
+                subtitle={intl.formatMessage({
+                  id: 'dashboard.secretsModal.annotations.error.subtitle',
+                  defaultMessage: getErrorMessage(errorMessage)
```

Comment:
If this is an error coming back from the API it shouldn't be processed using react-intl as we have no way of knowing the value in advance.

`subtitle={getErrorMessage(errorMessage)}` is fine here